python/openinference-instrumentation/src/openinference/instrumentation/__init__.py

```diff
@@ -12,11 +31,150 @@ def suppress_tracing() -> Iterator[None]:
             # No tracing will occur within this block
             ...
     """
-    try:
-        from opentelemetry.context import _SUPPRESS_INSTRUMENTATION_KEY, attach, detach, set_value
-    except ImportError:
-        yield
+
+    def __enter__(self) -> "suppress_tracing":
+        self._token = attach(set_value(_SUPPRESS_INSTRUMENTATION_KEY, True))
+        return self
+
+    def __exit__(self, exc_type, exc_value, traceback) -> None:
+        """Raise any exception triggered within the runtime context."""
+        detach(self._token)
+        return None
```

Comment:
Not sure this is doing much.

Reply:
You're correct, it was remanente from before